License, issue templates, contribution guide v2, other minor adjustments

isaac-kellogg-quantellia opened the pull request:
1:
Created MIT license file.
This will show on the home page for this repo once LICENSE is merged to main.

2:
Created issue templates. These will be identical to templates in the other repos.
See a2900bb for details.
Templates will activate once these files are merged to main.
Relevant issue labels [have been added](https://github.com/opendi-org/landing-site/labels).

Any updates to these issue templates will need to be manually mirrored to the other repos.

3:  
Updated contribution guide based on Nadine's feedback.
See 3a2a06e for details.

4:
Wording and format fixes:
- Clearer frontpage links
- Avoid overloading 'assets' term
- Broken links etc.

isaac-kellogg-quantellia committed:
Create LICENSE

Added MIT license from built-in GitHub template.

I set these parameters:
Year: 2024
Name: OpenDI

isaac-kellogg-quantellia committed:
Wording: 'assets' -> 'resources'

Reduced instances where the word 'asset' referred to OpenDI materials/resources, and not explicitly to Decision Assets,
to avoid overloading the term.

isaac-kellogg-quantellia committed:
Created issue templates

Added 3 issue templates:
- Bug Report
- Enhancement Request
- Other Issue

These help pre-sort issues before they have been triaged by automatically assigning labels.
They also provide structure and set expectations for what sorts of information we're looking for.

'Other Issue' is a catch-all for issues that don't sort into other templates. Asks contributors why the other templates were insufficient, in case we need to make new templates.

Also set issue template configuration to hide the "blank" template. This ensures all contributions are tagged and sorted by one of these 3 categories.

These templates will activate when these files are merged to `main`.

isaac-kellogg-quantellia committed:
Frontpage: Clearer resource links
Adopted `Learn More >>` style convention for main resource links, instead of having headers as links.

Also added another "drafts" qualifier to a reference to "standards".

isaac-kellogg-quantellia committed:
Fixed broken link

Link was given relative to main directory, needs relative link instead.

isaac-kellogg-quantellia committed:
Update contribution guide from round of feedback

Improved wording in a few places.
Updated images to be generally more clear.
Added additional Discord instructions for how to get to the right channels.
Added additional instructions for using Issue Templates.

ProfDavidRoberts commented on .github/ISSUE_TEMPLATE/bug-report.md:
```diff
@@ -0,0 +1,46 @@
+> Which category best fits this bug? Please indicate one of the following:
+> - **Content**: Bug concerns the standards content itself (inconsistency, typo, unclear, etc.)
+> - **Site**: Bug concerns the public website (broken link, error 404/missing content, etc.)
+> - **Deployment**: Bug concerns the deployment process (GitHub Actions, mkdocs generation process or configuration, etc.)
```
This won't hold up the approval, but we may want to consider editing this section to have repo-specific deployment tool chains (e.g., MKDocs vs. others).

ProfDavidRoberts commented on mkdocs.yml:
```diff
@@ -16,7 +16,7 @@ nav:
     - '(Working Draft) JSON Schema': https://opendi-org.github.io/json-schema
```
Should this point to opendi.org or GitHub.io?

isaac-kellogg-quantellia replied:
In testing, I found that these end up resolving to their corresponding `opendi.org` URLs anyway, so I haven't been strict about replacing them yet, especially since we're still awaiting certs for a couple of them.

Went ahead and changed this to the `http` opendi.org ones for now anyway, since that's at least more consistent than this. See c0cb888

ProfDavidRoberts commented on docs/index.md:
```diff
@@ -1,5 +1,5 @@
 # Welcome to OpenDI 
-We're glad you're here!  The purpose of this website is to publish OpenDI standards and assets for you to use and contribute to.  
+We're glad you're here!  The purpose of this website is to publish OpenDI standards and resources for you to use and contribute to.  
```
Suggest you replace 'standards and resources' with 'standard, materials, and resources'.

isaac-kellogg-quantellia replied:
Done in a0cc2c7

isaac-kellogg-quantellia committed:
MKDocs config: Replace deprecated links

Links using opendi-org.github.io replaced with their equivalent [subdomain].opendi.org links

isaac-kellogg-quantellia committed:
Homepage: Minor wording adjustment

'standards and resources' -> 'standards, materials, and resources'

ProfDavidRoberts reviewed (approved):
Looks great!
